src/navigation/wallet/screens/wallet-settings/ExportWallet.tsx

```diff
@@ -61,7 +62,7 @@ const PasswordActionContainer = styled.View`
 const PasswordInputContainer = styled.View`
   margin: 15px 0;
 `;
-
+/* add transalation */
```

Comment:
yes that comment is a TODO, where those comments are, a small refactor must be done to be able to use the hook. But that refactor should go in a different PR to avoid conflicts. Many files are being modified in this PR